python/src/trezorlib/nostr.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2025 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+
+from typing import TYPE_CHECKING
+
+from . import messages
+from .tools import expect
+
+if TYPE_CHECKING:
+    from .client import TrezorClient
+    from .protobuf import MessageType
+    from .tools import Address
+
+
+def get_pubkey(
+    client: "TrezorClient",
+    n: "Address",
+) -> str:
+    return client.call(
+        messages.NostrGetPubkey(
+            address_n=n,
+        )
+    ).pubkey.hex()
+
+
+@expect(messages.NostrEventSignature)
```

Comment:
per #4490 please drop the `@expect` and replace with `client.call(..., expect=NostrEventSignature)`
dtto for `get_pubkey`

Reply:
https://github.com/trezor/trezor-firmware/pull/4436/commits/a95dc04d5f42c357a5d00b171c1d780782b55f62#diff-d0a53275e15314e83ef900790cfdd6af05df702377562408a76287ae1507a853R36